test/utils/index.ts

```diff
@@ -13,4 +15,309 @@ describe('utils', () => {
 			assert.equal(value, '	\r\n\t svelte content');
 		});
 	});
+
+	describe('decode_character_references', () => {
```

Comment:
In terms of testing architecture, this file needs to move to `src/compiler/parse/utils/__test__.ts`.

Reply:
Sorry, I have mistaken the positon of test unit.I will fix it.